frontend/src/main/components/Users/UsersTable.js

```diff
@@ -49,21 +79,30 @@ export default function UsersTable({ users}) {
             Header: 'Admin',
             id: 'admin',
             accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
+            //accessor: (row, _rowIndex) => <span data-testid={`admin-${row.id}`}>{String(row.admin)}</span> 
         },
         {
             Header: 'Driver',
             id: 'driver',
             accessor: (row, _rowIndex) => String(row.driver) // hack needed for boolean values to show up
+            //accessor: (row, _rowIndex) => <span data-testid={`driver-${row.id}`}>{String(row.driver)}</span>
+        },
+        {
+            Header: 'Rider',
+            id: 'rider',
+            accessor: (row, _rowIndex) => String(row.rider) // hack needed for boolean values to show up
+            //accessor: (row, _rowIndex) => <span data-testid={`driver-${row.id}`}>{String(row.driver)}</span>
         }
     ];
-
+   
     const buttonColumn = [
         ...columns,
         ButtonColumn("toggle-admin", "primary", toggleAdminCallback, "UsersTable"),
+        ButtonColumn("toggle-driver", "primary", toggleDriverCallback, "UsersTable"),
+        ButtonColumn("toggle-rider", "primary", toggleRiderCallback, "UsersTable"),
     ]
-
     //const columnsToDisplay = showButtons ? buttonColumn : columns;
```

Comment:
No commented out code please.   Even if it's in the file that was given to you,  it's important to not leave it there.   

As they say among outdoors people: Leave the campsite cleaner than you found it.

The same applies to source code.  If there's something egregious in the file that you are editing, clean it up as you go.

Reply:
removed